qemu/tests/numa_opts.py

```diff
@@ -29,17 +30,19 @@ def run(test, params, env):
                 (len(numa), numa_nodes))
 
     for nodenr, node in enumerate(numa):
-        size = params.get("numa_node%d_size" % (nodenr))
-        if size is not None:
-            size = int(size)
-            if size != numa[nodenr][0]:
-                test.fail(
-                    "Wrong size of numa node %d: %d. Expected: %d" %
-                    (nodenr, numa[nodenr][0], size))
-
-        cpus = params.get("numa_node%d_cpus" % (nodenr))
+        mdev = params.get("numa_memdev_node%d" % (nodenr))
+        if mdev:
+            mdev = mdev.split('-')[1]
+            size = float(normalize_data_size(params.get("size_%s" % mdev)))
+        else:
+            size = params.get_numeric("mem")
```

Comment:
Eh~ It is an extra check for ppc nodes.0, the old design does not check this.  Could you explain it in the commit info ?   The method is ok for me, since without memdev,  the numa node size is the mem size of the vm. Thanks :)

Reply:
WM still has 1 default node even if there's no numa is specified  on ppc ,  thanks.

Reply:
> WM still has 1 default node even if there's no numa is specified on ppc , thanks.

Yes, automation code match with that, no problem. and the check method in this patch is ok too.
My point is, the old design does not check the size of the default numa for case `nodes.0` on ppc, and now it check this,  we can explain this in the commit info. thanks!

Reply:
@MiriamDeng Would you please provide test results on ppc?  Does it work? If not, I would remove this step to keep consistent with before. Thanks.

Reply:
(1/5) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.ppc64le.io-github-autotest-qemu.numa_opts.nodes.0: PASS (14.22 s)
 (2/5) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.ppc64le.io-github-autotest-qemu.numa_opts.nodes.1: PASS (15.62 s)
 (3/5) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.ppc64le.io-github-autotest-qemu.numa_opts.nodes.2: PASS (15.41 s)
 (4/5) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.ppc64le.io-github-autotest-qemu.numa_opts.nodes.3: PASS (15.51 s)
 (5/5) Host_RHEL.m8.u3.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.3.0.ppc64le.io-github-autotest-qemu.numa_opts.nodes.128: PASS (132.86 s)